### DISCUSSION TITLE
After last update the copy link doesn't work correctly

### DISCUSSION THREAD

**Discussion by grigoris79** [aclap-dev/video-downloadhelper#106] [Help and Q&A]
### Your question

What happened???
After the last update the copy url/link DOESN'T WORK CORRECTLY from the same sites before the last update work fine.

What really happened?????

I use the mozilla firefox addon.

**Comment by a-f-zz** [comment]
Version 8.1.4.0 worked great. Version 8.2.0.23 sucks. Stop updating without my permission and breaking stuff. I WANT 8.1.4.0 BACK!

**Comment by madmach01** [comment]
> ### Your question
> What happened??? After the last update the copy url/link DOESN'T WORK CORRECTLY from the same sites before the last update work fine.
> 
> What really happened?????
> 
> I use the mozilla firefox addon.
> 
> ### Addon & CoApp version and details
> _No response_

Same as you, I am running Chrome and VDH is version 8.2.0.23. Yesterday I was trying to use the copy function and it stopped working and crashed Chrome. I restarted chrome and the app looked smaller (font size), I noticed that when I used the copy function it wasn't copying anything. So it looks like I'm not the only one that's having problems.

**Comment by paulrouget** [comment]
Hello. We are aware of that issue. A fix is coming soon!

**Comment by grigoris79** [comment]
Still wait for an update to fix this issue.......

**Reply by jcfrog** [reply]
It has been submitted, waiting for approval.

**Reply by priteshbbbb** [reply]
Thanks for your hard work, guys!
